src/lib/components/sections/navbar.svelte

```diff
@@ -112,44 +112,50 @@
           }
 
           h3 {
-            font-size: clamp(1.25rem, 1.1635rem + 0.3846vw, 1.625rem);
+            font-size: clamp(1.25rem, 1.2039rem + 0.2632vw, 1.625rem);
```

Comment:
None of the CSS sizing tags are used here because the Logo text scales to very specific px sizes